lib/active_model/serializer.rb

```diff
@@ -27,32 +29,62 @@ class Serializer
       )
     /x
 
-    class << self
-      attr_accessor :_attributes
-      attr_accessor :_attributes_keys
+    # Hashes contents of file for +_cache_digest+
```

Comment:
Sorry I didn't call this out better. I included it in the PR description for the change: https://github.com/rails-api/active_model_serializers/issues/1176